bedrock/mozorg/templates/mozorg/home/home-a.html

```diff
@@ -0,0 +1,280 @@
+{% endblock %}
+
+{% block body_id %}home-a{% endblock %}
+{% block body_class %}home-variant home-variant-a{% endblock %}
+
+{% block extra_meta %}
+<!-- validates bing webmaster tools -->
+<meta name="msvalidate.01" content="B7B177115A634927D608514DA17B2574">
+<!-- YouTube Verification -->
+<meta name="google-site-verification" content="U9a6gH32vLIykvntaDToj-ytYhlZ1AfAgVEKstixQIE">
+{% endblock %}
+
+{% block extrahead %}
+  {% stylesheet 'home-a' %}
+{% endblock %}
+
+{% block optimizely %}
+  {% if waffle.switch('mozorg-home-optimizely') %}
+    {% include 'includes/optimizely.html' %}
+  {% endif %}
+{% endblock %}
+
+{% block site_header %}
+<header class="section masthead" id="masthead">
+  <div class="content">
+
+  {% block tabzilla_tab %}{% endblock %}
+
+  {% block site_header_nav %}
+    <nav class="masthead-nav-main" id="nav-main">
+      <span class="toggle" role="button" aria-controls="nav-main-menu" aria-expanded="false" tabindex="0">{{ _('Menu') }}</span>
+      <ul id="nav-main-menu">
+        <li class="first about-item">
+          <a href="{{ url('mozorg.about') }}" data-link-type="nav" data-link-name="About">{{ _('About') }}</a>
+        </li>
+        <li class="contribute-item">
+          <a href="{{ url('mozorg.contribute') }}" data-link-type="nav" data-link-name="Participate">{{ _('Participate') }}</a>
+        </li>
+        <li class="products-item">
+          <a href="{{ url('firefox.family.index') }}" data-link-type="nav" data-link-name="Firefox">{{ _('Firefox') }}</a>
+        </li>
+      </ul>
+    </nav>
+  {% endblock %}
+
+  {% block site_header_logo %}
+    <h2 class="masthead-logo">
+      <a href="#intro">
+        <img src="{{ static('img/home/new/mozilla-wordmark.svg') }}" alt="Mozilla">
+      </a>
+    </h2>
+  {% endblock %}
+
+  <nav class="masthead-nav-page" id="nav-page">
+    <ul id="nav-page-primary" class="primary">
+      <li id="nav-primary-action"><a href="#action">{{ _('Mozilla In Action') }}</a></li>
+      <li id="nav-primary-innovation"><a href="#innovation">{{ _('Web Innovations') }}</a></li>
+      <li id="nav-primary-firefox"><a href="#firefox">{{ _('Firefox') }}</a></li>
+    </ul>
+
+    <ul id="nav-page-secondary" class="secondary">
+      <li id="nav-secondary-donate"><a href="{{ donate_url('home_test_a') }}">{{ _('Donate') }}</a></li>
+      <li id="nav-secondary-firefox">
+        {{ download_firefox(alt_copy=_('Download Firefox'), button_color='transparent', dom_id='nav-download-firefox') }}
+      </li>
+    </ul>
+  </nav>
+
+  </div>
+</header>
+{% endblock %}
+
+{% block content %}
+<main role="main" class="main">
+  <section class="section section-intro" id="intro">
+    <div id="home-slideshow">
+      <div class="slide" id="slide-one"></div>
+      <div class="slide" id="slide-two"></div>
+      <div class="slide" id="slide-three"></div>
+    </div>
+    <div class="content">
+      <h1 class="intro-tagline">
```

Comment:
For SEO purposes (bleugh), should our logo be the `<h1>` on this page, as opposed to this tagline?